addons/mail/static/src/messaging/component_hook/use_store/use_store.js

```diff
@@ -117,7 +117,7 @@ function useStore(selector, options = {}) {
 
     const extendedSelector = (state, props) => selector(props);
     return owl.hooks.useStore(extendedSelector, Object.assign({}, options, {
-        isEqual: proxyComparatorDeep(options.compareDepth),
+        // isEqual: proxyComparatorDeep(options.compareDepth),
```

Comment:
basically makes this useStore equivalent than original OWL useStore without `state` as 1st parameter